docs/advanced-topics/custom-serialization.md

```diff
@@ -13,7 +13,9 @@ That is, when Netcode first gets hold of a type, it will check for any custom ty
 
 By default, any type that satisfies the `unmanaged` generic constraint can be automatically serialized as RPC parameters. This includes all basic types (bool, byte, int, float, enum, etc) as well as any structs that contains only these basic types.
 
-With this flow, you can override **ALL** serialization for **ALL** types, even built in types, and with the API provided, it can even be done with types that you have not defined yourself, those who are behind a 3rd party wall, such as .NET types.
+With this flow, you can override **ALL** serialization for **ALL** types, even built in types, and with the API provided, it can even be done with types that you have not defined yourself, those who are behind a 3rd party wall, such as .NET types. However, the way custom serialization is implemented for RPCs and NetworkVariables is slightly different.
```

Comment:
```suggestion
With this flow, you can override **all** serialization for **all** types, even built-in types, and with the API provided, it can even be done with types that you have not defined yourself, those who are behind a 3rd party wall, such as .NET types. However, how custom serialization is implemented for RPCs and NetworkVariables is slightly different.
```

Reply:
I feel like "the way this is implemented" is more grammatically correct (or at least sounds more natural to me) than "how this is implemented", in this context...

Reply:
Made a commit that replaced that whole first sentence, because it wasn't really accurate anyway.